src/containers/Movie/Movie.js

```diff
@@ -4,13 +4,15 @@ import { connect } from 'react-redux';
 
 const Movie = ({ id, title, poster, rating, overview }) => {
   return (
-    <section>
-      <h3>{title}</h3>
-      <p>{rating}</p>
```

Comment:
Curious if we will want rating in there, but let's see once we get more info  showing